docs/data.mdx

```diff
@@ -0,0 +1,272 @@
+
+```javascript
+{
+  "tracks":[{
+    "data": {
+        "url": "https://resgen.io/api/v1/tileset_info/?d=UvVPeLHuRDiYA3qwFlm7xQ",
+        "type": "multivec",
+        "row": "sample",
+        "column": "position",
+        "value": "peak",
+        "categories": ["sample 1", "sample 2", "sample 3", "sample 4"]
+    },
+    ...// other configurations of this track
+  }]
+}
+```
+
+<TableWrapper objName='MultivecData'/>
+
+ 
+### BEDDB (Require HiGlass Server)
+
+Regular BED, or similar, files can be pre-aggregated for the scalable data exploration. Find our more about this format at [HiGlass Docs](https://docs.higlass.io/data_preparation.html#bed-files).
+
+```javascript
+{
+  "tracks":[{
+    "data": {
+      "url": "https://higlass.io/api/v1/tileset_info/?d=OHJakQICQD6gTD7skx4EWA",
+      "type": "beddb",
+      "genomicFields": [
+          {"index": 1, "name": "start"},
+          {"index": 2, "name": "end"}
+      ],
+      "valueFields": [
+          {"index": 5, "name": "strand", "type": "nominal"},
+          {"index": 3, "name": "name", "type": "nominal"}
+      ],
+      "exonIntervalFields": [
+          {"index": 12, "name": "start"},
+          {"index": 13, "name": "end"}
+      ]
+    },
+    ... // other configurations of this track
+  }]
+}
+```
+
+<TableWrapper objName='BEDDBData'/>
+
+
+
+## Data Transform
+
+<span>
+    Gosling supports a diverse set of data transforms, including 
+    {GoslingSchema["definitions"]['DataTransform'].anyOf.map((d, i)=>{
+      let name = d.$ref.replace('#/definitions/', '')
+      name = name.replace(/([a-z])([A-Z])/g, '$1 $2').replace(/([A-Z])([A-Z][a-z])/g, '$1 $2')
+        return <span key={name}> <a href={`#${name.replace(' ', '-').toLowerCase()}`}>
+            {name}
+        </a> {i==GoslingSchema["definitions"]['DataTransform'].anyOf.length-1 ? '. ': ', '} </span>})
+    }
+</span>
+
+
+```javascript
+{
+  "tracks":[{
+    "data": ...,
+    // a list of data transforms can be applied to the data
+    "dataTransform": [
+          { "type": "filter", "field": "type", "oneOf": ["gene"] },
+          { "type": "filter", "field": "strand", "oneOf": ["+"], "not": true }
+    ],
+    "mark": "rect",
+    ...,
+  }]
+}
+```
+
+### Filter Transform
+
+Users can apply three types of filters: `oneOf`, `inRange`, `include`.
+Each filter transform has the following properties:
+
+| property | type                 | description                                                                          |
```

Comment:
Is there a reason only this part being a manual table? Or perhaps,

```
<TableWrapper objName='FilterTransform' includeDescription={true}/>
```

Reply:
I think the three auto tables are more understandable since those filter transforms require different properties and that is explained well in separate tables. What do you think?